src/DetailsView/components/reflow-assessment-view.tsx

```diff
@@ -21,6 +28,7 @@ export interface ReflowAssessmentViewProps {
     currentTarget: Tab;
     prevTarget: PersistedTabInfo;
     assessmentTestResult: AssessmentTestResult;
+    isEnabled: boolean;
```

Comment:
What is `isEnabled` referring to here? This might be a good opportunity to re-factor/rename a property that was ambiguous in the old assessment view.

Reply:
isEnabled refers to whether or not the currently selected test requirement is enabled. I agree that it's ambiguous, but I'd prefer to change it in a seperate pr.